datadog/api/base.py

```diff
@@ -124,6 +124,14 @@ def request(cls, method, path, body=None, attach_host_name=False, response_forma
                 if e.response.status_code == 404 or e.response.status_code == 400:
                     pass
```

Comment:
403 should probably be caught in this block and raise at this point https://github.com/DataDog/datadogpy/blob/clarify_mute_all/datadog/api/base.py#L160